### PR TITLE
Update release docs with constraints file release steps

### PR DESCRIPTION
**Pull request by pankajastro**
# Description
closes: https://github.com/astronomer/astro-sdk/issues/1474
## What is the current behavior?
Update release docs with constraints file release steps

## Does this introduce a breaking change?
No

### Checklist
- [ ] Created tests which fail without the change (if possible)
- [ ] Extended the README / documentation, if necessary

### CODE REVIEW

File: python-sdk/docs/development/RELEASE.md [hunk 1]
```diff
@@ -72,3 +72,29 @@ target the release branch (e.g. `release-0.7`). Failure to do these steps proper
 <!-- markdown-link-check-enable -->
 
 Once you've created your release, that's it! The rest of the steps should be handled by the CI system.
+
+# Releasing constraints for the release
```
Comment (kaxil) [inline python-sdk/docs/development/RELEASE.md:76]: Did you also verify all the steps by running them once in a dummy branch?

Comment (pankajastro) [reply python-sdk/docs/development/RELEASE.md:76]: thanks for pointing it out. except `git tag constraints-1.4.0 && git push origin constraints-1.4.0` I have tested other command after your comment

File: python-sdk/docs/development/RELEASE.md [hunk 2]
```diff
@@ -72,3 +72,35 @@ target the release branch (e.g. `release-0.7`). Failure to do these steps proper
...
+   ```console
+   ❯ unzip "*.zip" &&  rm -rf *.zip
+   ```
+5. Then rename them
+   ```console
+   ❯ for f in constraints*; do mv "$f" "$f.txt"; done
+   ```
+6. Add and upload
+   ```console
+   ❯ git add const* && git commit -m "Update constraints for Astro SDK 1.4.0" --no-verify  && git push origin constraints-1-4 && git tag constraints-1.4.0 && git push origin constraints-1.4.0
```
Comment (kaxil) [inline python-sdk/docs/development/RELEASE.md:105]:
```suggestion
   git add const*
   git commit -m "Update constraints for Astro SDK 1.4.0" --no-verify  
   git push origin constraints-1-4 
   git tag constraints-1.4.0 
   git push origin constraints-1.4.0
```

File: python-sdk/docs/development/RELEASE.md [hunk 3]
```diff
@@ -72,3 +72,35 @@ target the release branch (e.g. `release-0.7`). Failure to do these steps proper
...
+   ```console
+   ❯ rm *.txt
+   ```
+4. Find the GitHub action CI job that ran when you tagged the release and which published the artifact i.e. visit https\://github.com/astronomer/astro-sdk/actions/runs/<RUN_ID> and form the bottom of that GH page download all the constraints file and unzip it.
+   ```console
+   ❯ unzip "*.zip" &&  rm -rf *.zip
+   ```
+5. Then rename them
+   ```console
+   ❯ for f in constraints*; do mv "$f" "$f.txt"; done
```
Comment (kaxil) [inline python-sdk/docs/development/RELEASE.md:101]:
```suggestion
   for f in constraints*; do mv "$f" "$f.txt"; done
```

File: python-sdk/docs/development/RELEASE.md [hunk 4]
```diff
@@ -72,3 +72,35 @@ target the release branch (e.g. `release-0.7`). Failure to do these steps proper
...
+   ```console
+   ❯ git checkout constraints-1-4
+   ```
+3. Remove old files
+   ```console
+   ❯ rm *.txt
+   ```
+4. Find the GitHub action CI job that ran when you tagged the release and which published the artifact i.e. visit https\://github.com/astronomer/astro-sdk/actions/runs/<RUN_ID> and form the bottom of that GH page download all the constraints file and unzip it.
+   ```console
+   ❯ unzip "*.zip" &&  rm -rf *.zip
```
Comment (kaxil) [inline python-sdk/docs/development/RELEASE.md:97]:
```suggestion
   unzip "*.zip" &&  rm -rf *.zip
```

File: python-sdk/docs/development/RELEASE.md [hunk 5]
```diff
@@ -72,3 +72,35 @@ target the release branch (e.g. `release-0.7`). Failure to do these steps proper
...
+   ```console
+   ❯ git branch constraints-1-4 constraints-1-3
+   ```
+2. Checkout that branch
+   ```console
+   ❯ git checkout constraints-1-4
+   ```
+3. Remove old files
+   ```console
+   ❯ rm *.txt
```
Comment (kaxil) [inline python-sdk/docs/development/RELEASE.md:93]:
```suggestion
   rm *.txt
```

File: python-sdk/docs/development/RELEASE.md [hunk 6]
```diff
@@ -72,3 +72,35 @@ target the release branch (e.g. `release-0.7`). Failure to do these steps proper
...
+1. First create a new branch of the last constraints branch.
+   ```console
+   ❯ git fetch origin constraints-1-3 && git checkout constraints-1-3
+   ```
+   ```console
+   ❯ git branch constraints-1-4 constraints-1-3
+   ```
+2. Checkout that branch
+   ```console
+   ❯ git checkout constraints-1-4
```
Comment (kaxil) [inline python-sdk/docs/development/RELEASE.md:89]:
```suggestion
   git checkout constraints-1-4
```

File: python-sdk/docs/development/RELEASE.md [hunk 7]
```diff
@@ -72,3 +72,35 @@ target the release branch (e.g. `release-0.7`). Failure to do these steps proper
...
+# Releasing constraints for the release
+
+Once the package pushed to PyPI/Readthedocs you also need to release constraints files. This steps it bit manual at this point of time.
+
+1. First create a new branch of the last constraints branch.
+   ```console
+   ❯ git fetch origin constraints-1-3 && git checkout constraints-1-3
+   ```
+   ```console
+   ❯ git branch constraints-1-4 constraints-1-3
```
Comment (kaxil) [inline python-sdk/docs/development/RELEASE.md:85]:
```suggestion
   git branch constraints-1-4 constraints-1-3
```

File: python-sdk/docs/development/RELEASE.md [hunk 8]
```diff
@@ -72,3 +72,35 @@ target the release branch (e.g. `release-0.7`). Failure to do these steps proper
...
 
 Once you've created your release, that's it! The rest of the steps should be handled by the CI system.
+
+# Releasing constraints for the release
+
+Once the package pushed to PyPI/Readthedocs you also need to release constraints files. This steps it bit manual at this point of time.
+
+1. First create a new branch of the last constraints branch.
+   ```console
+   ❯ git fetch origin constraints-1-3 && git checkout constraints-1-3
```
Comment (kaxil) [inline python-sdk/docs/development/RELEASE.md:82]:
```suggestion
   git fetch origin constraints-1-3 && git checkout constraints-1-3
```

**Review by kaxil — APPROVED** [review]

**Pull request merged**